src/main/resources/assets/mekanism/lang/es_es.lang

```diff
@@ -330,445 +330,444 @@ tile.MachineBlock3.FuelwoodHeater.name=Fuelwood Heater
 
 # Plastic
 
-tile.PlasticBlock.name=Plastic Block
+tile.PlasticBlock.name=Bloque de Plastico
 
-tile.SlickPlasticBlock.name=Slick Plastic Block
+tile.SlickPlasticBlock.name=Bloque de Plastico Manchado
 
-tile.GlowPlasticBlock.name=Plastic Glow Block
+tile.GlowPlasticBlock.name=Bloque de Plastico Luminoso
 
-tile.ReinforcedPlasticBlock.name=Reinforced Plastic Block
+tile.ReinforcedPlasticBlock.name=Bloque de Plastico Reforzado
 
-tile.RoadPlasticBlock.name=Plastic Road
+tile.RoadPlasticBlock.name=Vara de Plastico
 
 
 
-tile.PlasticFence.name=Plastic Barrier
+tile.PlasticFence.name=Valla de Plastico
 
 
 
 # Infuse types
 
 infuse.carbon=Carbon
 
-infuse.tin=Tin
+infuse.tin=Estaño
 
-infuse.diamond=Diamond
+infuse.diamond=Diamante
 
 infuse.redstone=Redstone
 
-infuse.fungi=Fungi
+infuse.fungi=Hongos
 
-infuse.obsidian=Obsidian
+infuse.obsidian=Obsidiana
 
-infuse.bio=Biomass
+infuse.bio=Biomasa
 
 
 
 # Ore Block
 
-tile.OreBlock.OsmiumOre.name=Osmium Ore
+tile.OreBlock.OsmiumOre.nam= Mena de Osmio
```

Comment:
`e` got delete here!

Reply:
And a random space got added after the `=`